src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ CHANGELOG
 7.3
 ---
 
+ * Add `TreeHelper` to display tree-like structures
+ * Add `TreeStyle` to customize the output of the tree helper
+ * Add `createTree()` methods to `SymfonyStyle`
```

Comment:
```suggestion
 * Add `SymfonyStyle::createTree()`
```
?

Reply:
I wonder if the two first entries should be merged. Usually, we're not that descriptive in the Changelog. Should it be maybe:

```suggestion
 * Add `TreeHelper` and `TreeStyle` to display tree-like structures
 * Add `SymfonyStyle::createTree()`
```

Reply:
Done, thanks!